Algorithms/Strings/lcc.cs

```diff
@@ -1,11 +1,10 @@
 using System;
 
-namespace Example
+namespace Algorithms.Strings
 {
-    public static partial class StringViewModel
+    public static class StringViewModel
```

Comment:
```suggestion
    public static class GeneralStringAlgorithms
```
StringViewModel sounds strange to me and doesn't give any clues on what this class is about. I suggest renaming it to something more descriptive. Maybe GeneralStringAlgorithms if we put other simple string algorithms in here in future.